crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -708,7 +708,7 @@ impl<'w, T: Fetch<'w>> Fetch<'w> for OptionFetch<T> {
 ///         }
 ///     }
 /// }
-/// # print_moving_objects_system.system();
+/// # print_moving_objects_system;
```

Comment:
This usage in doc-tests shouldn't be removed, as it checks if this function is still a valid system.
This check is also done in many other doc-tests.

Reply:
oh forgot that pattern in bevy_ecs... thanks, I added them back